web/src/app/administration/components/access-items-management/access-items-management.component.spec.ts

```diff
@@ -186,7 +186,7 @@ describe('AccessItemsManagementComponent', () => {
     });
   }));
 
-  it('should display a dialog in when access is revoked', async(() => {
+  it('should display a dialog when access is revoked', async(() => {
```

Comment:
this may conflict with the other PR since both have changes in access-items-management.component.spec.ts